src/twisted/trial/test/detests.py

```diff
@@ -150,7 +150,8 @@ def test_expectedFailure(self):
         d = defer.succeed('todo')
         d.addCallback(self._cb_error)
         return d
-    test_expectedFailure.todo = "Expected failure"
+    test_expectedFailure.todo = "Expected failure"  # type: ignore[attr-defined]  # noqa
```

Comment:
I wonder if some `@todo`, `@skip`, etc. decorators would make this ness stupid.

I've been doing the same, but there are a *lot* of these in Twisted.

Reply:
Well in other PR's I moved all the tests to use the @skipIf decorator from the Python unittest core library.
Implementing a @todo decorator in Twisted may not be a bad idea, and would be a good way to clean these up without adding comments specific to mypy